src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -408,7 +412,8 @@ public void run() {
 				
 				arrested.removeWanted();
 				
-				// TODO warn arrested
+				MessageLanguageManager messages = MessageLanguageManager.getInstance(arrested.getName());
```

Comment:
This method receives as argument the language. Use getInstanceByPlayer method instead. (I am not sure if the method name is exactly that).